### ISSUE TITLE
Error on /modreq status

### ISSUE THREAD

**Issue by CloudeLecaw** [SuperRonanCraft/AdvancedModreq#6]
If you Just use /modreq Status withut any playername you get this error:

```
[16:25:36] [Server thread/INFO]: Cloude_Lecaw issued server command: /modreq status
[16:25:36] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'modreq' in plugin AdvancedModreq v5.5.1
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:696) ~[spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1644) ~[spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1479) ~[spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.a(SourceFile:10) ~[spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:?]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:?]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:900) [spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:835) [spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:733) [spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	at java.lang.Thread.run(Unknown Source) [?:?]
Caused by: java.lang.ArrayIndexOutOfBoundsException: 1
	at me.SuperRonanCraft.AdvancedModreq.events.Commands.status(Commands.java:295) ~[?:?]
	at me.SuperRonanCraft.AdvancedModreq.events.Commands.onCommand(Commands.java:129) ~[?:?]
	at me.SuperRonanCraft.AdvancedModreq.Main.onCommand(Main.java:140) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot-1.13.1.jar:git-Spigot-f823ac5-5cffeca]
	... 15 more
```

**Comment by SuperRonanCraft** [comment]
Sorry for the late delay, I did see your issue, just catching up with all my other project :)

I do see that this command is not actually functional, it only works at the moment with a player name after the command, I will be changing this to not require a players name after and to check if the player is valid when one is typed in.
Such as
"/modreq status" views your own ticket status'
and "/modreq status <player>" can let you view another players tickets status

**Comment by SuperRonanCraft** [comment]
Fixed on 5.5.2
